server/item/elytra.go

```diff
@@ -0,0 +1,57 @@
+package item
+
+import "github.com/df-mc/dragonfly/server/world"
+
+// Elytra is a pair of rare wings found in end ships that are the only single-item source of flight in Survival mode.
+type Elytra struct{}
+
+// Use handles the using of an elytra to auto-equip it in an armour slot.
+func (Elytra) Use(_ *world.World, _ User, ctx *UseContext) bool {
+	ctx.SwapHeldWithArmour(1)
+	return false
+}
+
+// DurabilityInfo ...
+func (Elytra) DurabilityInfo() DurabilityInfo {
+	return DurabilityInfo{
+		MaxDurability: 433,
+		Persistent:    true,
+		BrokenItem:    simpleItem(Stack{}),
```

Comment:
Have you tried setting the BrokenItem to an Elytra with 0 durability? That way we might not need this extra Persistent bool.

Reply:
It'll still play the break sound etc no?

Reply:
That'd be reasonable though, no? Only thing I'm worried about is that maybe it'll be removed from its slot, but you could test that I guess?

Reply:
The sound doesn't play in vanilla I don't think, it just instantly prevents you from gliding

Reply:
Removed.